internal/locate/region_request.go

```diff
@@ -1344,6 +1345,9 @@ func (s *RegionRequestSender) sendReqToRegion(
 		req.ForwardedHost = rpcCtx.Addr
 		sendToAddr = rpcCtx.ProxyAddr
 	}
+	if req.StoreTp == tikvrpc.TiKVRemoteCoprocessor {
```

Comment:
Is there only one remote cop worker address for each request?

Reply:
Yes.